data/remnant missions.txt

```diff
@@ -495,3 +565,691 @@ mission "Remnant: Technology Available"
+			label end
+			`	"Well, I need to get back to work. That last battle put us behind schedule on our repairs. Have a good day!" With a final flourish that you think is the equivalent of an exclamation mark, Tali strides out of the outfitters, heading towards another landing bay.`
+
+event "Remnant Salvage Available"
+	planet "Aventine"
+		outfitter "Remnant Salvage"
+	planet "Caelian"
+		outfitter "Remnant Salvage"
+	planet "Viminal"
+		outfitter "Remnant Salvage"
+
+
+
+mission "Remnant: Bounty 2"
+	job
+	repeat
+	name "Remnant bounty"
+	description "Hunt down a Korath ship that is lurking in Remnant territory, then return to <planet> to receive your payment of <payment>."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Salvage 4: offered"
+		random < 60
+	npc kill
+		government "Korath"
+		personality heroic vindictive target uninterested waiting
+		system
+			distance 1 2
+		fleet
+			names "korath"
+			cargo 3
+			variant
+				"Korath Raider (Hyperdrive)"
+		dialog "You have destroyed the Korath ship. You can now return to <planet> to collect your payment."
+	on complete
+		payment 300000
+		dialog "A Remnant military leader thanks you for hunting down the Korath ship <npc>, and gives you the agreed-upon payment of <payment>."
+
+
+
+mission "Remnant: Expanded Horizons Quarg 1"
+	name "Visiting the Quarg"
+	description "Take Remnant researchers to <destination> to see the Quarg."
+	minor
+	source
+		government "Remnant"
+	passengers 3
+	destination "Wayfarer"
+	to offer
+		has "Remnant: Learn Sign Follow Up: offered"
+		has "Remnant: Salvage 1: done"
+		has "First Contact: Quarg: offered"
+		random < 40
+	on offer
+		conversation
+			`As you relax in the cafeteria on <origin> you are approached by a young Remnant. Judging by the hand-held sensors and tools hanging off her belt, you suppose she is a researcher or an engineer.`
+			`	"Hello Captain <last>. My name is Dawn, and I have been spending the past few days skimming the archive you retrieved. I wanted to ask you a few questions. May I?"`
+			`	She slides into the chair opposite you without waiting for your reply, and pulls out a datapad. "Firstly, our histories mention how humanity met a race called the 'Quarg' and had some ongoing dialogs with them, but not much beyond that. Now here in this history that you brought us, it says that humanity has started exchanging technology with them! Is this true?" She looks at you intently. You sign negatively, and she dejectedly continues. "Could you explain about them, then?" You reply that the Quarg have allowed humanity to live near them, and that the Quarg are usually willing to answer questions, including scientific ones. You continue, elaborating that while the Quarg don't seem to mind having an innovative shipyard studying them as best it can, they don't actually share their technology with anyone.`
```

Comment:
It's a bit of a stretch to say the Quarg are willing to answer scientific questions. They are to some extent, but they likely don't answer questions that could lead us to new, dangerous technology and don't help us with our research. Otherwise figuring out JDs would be a simple task, right?

Reply:
That's more or less common sense. Walking up and asking "how do skylances work" is the same thing (even more effective, actually) as stealing one to reverse engineer it. Why would the quarg prohibit one but not the other?

Reply:
>they did make their starting money in a syndicate factory. 

*checks map*
New Boston ain't in Syndicate space.

Reply:
I think them telling you how skylances work would come under "sharing", and they don't like you stealing them. (And it might just be too hard to reverse engineer anyway.)

Reply:
So, the actual lore would be that the Quarg do answer scientific questions, but they are very choosy about what they answer. In particular, they avoid answering questions that would lead directly to really dangerous things like Jump Drives and star-destroying weapons. And they definitely dislike people stealing their stuff. 

(This is one of the many places I'd love to have finer grained fines and legality. It'd be awesome if we could make it so that the Quarg fined (or even attacked) any ship that had Quarg outfits on it without having a Quarg License from them. But that's really outside the purview of this PR)